modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/rest-openapi.yaml

```diff
@@ -279,8 +279,8 @@ components:
                     readOnly: true
                     type: object
                 quantity:
-                    format: int32
-                    type: integer
+                    format: double
```

Comment:
sync with commits from COMMERCE-11747

Reply:
talked to @smottal, confirmed that his work on https://liferay.atlassian.net/browse/COMMERCE-11747, will include headless changes for updating quantity related fields to BigDecimal.